cmd/cosign/cli/verify/verify_blob.go

```diff
@@ -119,11 +119,16 @@ func VerifyBlobCmd(ctx context.Context, ko options.KeyOpts, certRef, certEmail,
 			EnforceSCT:                   enforceSCT,
 		}
 		if certChain == "" {
```

Comment:
We need to also update the documentation for the `--certificate` flag to note that it verifies the certificate against the Fulcio roots provided by TUF.

Reply:
Fixed https://github.com/sigstore/cosign/pull/2139/commits/b4887ea391a33cedd595d3b193baae5dc15d037e